src/util/merkleblock.rs

```diff
@@ -53,20 +55,45 @@ use crate::consensus::encode::{self, Decodable, Encodable};
 use crate::util::merkleblock::MerkleBlockError::*;
 use crate::{Block, BlockHeader};
 
-/// An error when verifying the merkle block
+/// An error when verifying the merkle block.
 #[derive(Clone, PartialEq, Eq, Debug)]
 #[non_exhaustive]
 pub enum MerkleBlockError {
-    /// When header merkle root don't match to the root calculated from the partial merkle tree
+    /// Merkle header root doesn't match to the root calculated from the partial merkle tree.
```

Comment:
Shouldn't this be "Merkle root in the header doesn't match..."? Never heard of "merkle header root".

Reply:
Sure thing, I have no idea. Will use as suggested.